src/connection_string.ts

```diff
@@ -1092,6 +1092,7 @@ export const OPTIONS = {
     type: 'string'
   },
   socketTimeoutMS: {
+    deprecated: 'Please use timeoutMS instead',
```

Comment:
Undo the deprecations since we decided to ship timeoutMS expiremental. We will deprecate later.

Reply:
Use NODE-6491 for the un-deprecation and tag a TODO to that ticket (mention the undeprecation in NODE-6491)